src/duct/migrator/ragtime.clj

```diff
@@ -59,8 +59,13 @@
   (update migration :id str "#" (subs (hash-migration migration) 0 8)))
 
 (defn- migrate [index {:keys [database migrations strategy logger]
-                       :or   {strategy :raise-error}}]
-  (ragtime/migrate-all (jdbc/sql-database (:spec database)) index migrations
+                       :or   {strategy :raise-error}
+                       :as   opts}]
+  (ragtime/migrate-all (jdbc/sql-database (:spec database)
+                                          (or (select-keys opts [:migrations-table])
```

Comment:
Can you put this in the argument destructuring instead?

Reply:
Sure I apply it.